common/laws/00_land_reform.txt

```diff
@@ -9,7 +9,9 @@
 	}
 	modifier = {
 		interest_group_ig_landowners_pol_str_mult = 0.5
+		country_military_goods_cost_mult = -0.10
 		tax_land_add = 0.1
+		state_birth_rate_mult = 0.01
```

Comment:
maybe accepted birth rate is better?  state_accepted_birth_rate_mult

Reply:
that's true, i'll correct that